ci/requirements-2.7.run

```diff
@@ -18,3 +18,4 @@ patsy
 pymysql=0.6.3
 jinja2=2.8
 xarray=0.8.0
+boto3
```

Comment:
I don't think you should have to add boto3 anywhere. It'll already be installed if we're using s3fs.

I would just add `moto` to every file with s3fs. See https://github.com/pandas-dev/pandas/issues/17325#issuecomment-324886492